tink/server/internal/grpc/kube_test.go

```diff
@@ -80,7 +80,7 @@ func TestMakeValid(t *testing.T) {
 		"valid name with prefix and truncated name": {
 			inputName:    "very-long-name-that-exceeds-the-sixty-three-character-limit-and-needs-to-be-truncated",
 			inputPrefix:  "enrollment-",
-			expectedName: "enrollment-very-long-name-that-exceeds-the-sixty-three-characte",
+			expectedName: "enrollment-very-long-name-that-exceeds-the-sixty-three-character",
```

Comment:
This actually needs to be spelled as is because the name gets truncated to 63 characters.

Reply:
Thanks, reverted.